recipes/r-msqc/meta.yaml

```diff
@@ -0,0 +1,70 @@
+{% set version = '1.0.2' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-msqc
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/MSQC_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/MSQC/MSQC_{{ version }}.tar.gz
+  sha256: f6f925845b977fd3073041911b7e43e15c162a257a6ba1c36196cacb08f08c87
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{posix}}zip               # [win]
+  host:
+    - r-base
+    - r-rgl
+  run:
+    - r-base
+    - r-rgl
+
+test:
+  commands:
+    - $R -e "library('MSQC')"           # [not win]
+    - "\"%R%\" -e \"library('MSQC')\""  # [win]
+
+about:
+  home: https://CRAN.R-project.org/package=MSQC
+  license: GPL-2 | GPL-3
+  summary: This is a toolkit for multivariate process monitoring. It computes several multivariate
```

Comment:
This can probably be `description`, and the first sentence can go under `summary`.